kernelci.org/content/en/docs/org/tsc/_index.md

```diff
@@ -55,14 +55,22 @@ Decisions that always require a vote are:
 * granting or removing any kind of admin rights to individuals
 * making proposals to the LF project board
 
-The result of each vote should be shared on the mailing list and added to the
-[voting records](votes).  It should also include the date when any related
-changes were enacted.  As such they provide a way to keep a formal trace of
-decisions being made.
-
-Votes done during a live meeting require 50% of the TSC members to be voting
-and a majority among these voting members.  Votes done by email or any other
-deferred channel require a majority among all of the TSC members.
+Votes are made on motions with a yes/no choice and have to be proposed by a TSC
```

Comment:
Can TSC members abstain from a vote?

Reply:
That's a very good point, I was wondering about this as it's a bit of a grey area.  Say, if there are 7 voting members in a meeting, 3 of them abstain and 3 vote "yes" (and one "no").  That's only 3/7 "yes", so if you take abstentions into account the vote outcome is "no" and abstentions are basically the same as voting "no".  If you don't take abstentions into account and just equate them to not voting, then 4 people are voting out of 7, there's still quorum and it's a 3/4 majority so the outcome is then "yes".

My inclination would be to not have abstentions, just people vote "yes" or "no" or don't vote at all.  So in the example above it would be a "yes".  I'd be interested to hear what others think of this, it's probably worth a TSC email topic or something for the monthly meeting.

Reply:
Also it would make sense to have common voting rules for the TSC and the board.  Once this is clarified for the TSC, the board page should probably be updated accordingly.

Reply:
Please see #298, I thought this should deserve a separate PR as it's a bit more open to debate than this current one.